ibm_catalog.json

```diff
@@ -22,48 +22,48 @@
             "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-cos/main/images/cos_icon.svg",
             "features": [
                 {
-                    "title": "Create Object Storage instance",
+                    "title": "Create an Object Storage instance",
                     "description": "Create an IBM Cloud Object Storage instance."
                 },
                 {
-                    "title": "Create Object Storage bucket",
+                    "title": "Create an Object Storage bucket",
                     "description": "Create either a cross-regional or regional Object Storage bucket."
                 },
                 {
-                    "title": "Create HMAC key",
+                    "title": "Create an HMAC key",
                     "description": "Create an HMAC key for the Object Storage instance."
                 },
                 {
                     "title": "Attach access tags",
                     "description": "Attach access tags to the Object Storage instance."
                 },
                 {
-                    "title": "Bucket retention",
-                    "description": "Configure bucket retention."
+                    "title": "Configure bucket retention",
+                    "description": "Configure retention policies at the IBM Cloud Object Storage bucket level."
                 },
                 {
-                    "title": "Object versioning",
+                    "title": "Enable object versioning",
                     "description": "Supports enabling object versioning to keep multiple versions of an object in a bucket."
                 },
                 {
-                    "title": "Archiving",
+                    "title": "Add archive policies",
                     "description": "Add archive policies on the bucket."
                 },
                 {
-                    "title": "Activity Tracking",
-                    "description": "Integrate Activity Tracker events."
+                    "title": "Record Activity Tracker events",
+                    "description": "Integrate Activity Tracker to track events."
                 },
                 {
-                    "title": "Monitoring",
-                    "description": "Provision IBM Cloud Monitoring on Object Storage data."
+                  "title": "Monitor data",
+                  "description": "Provision IBM Cloud Monitoring on Object Storage data."
```

Comment:
This description is still wrong. It should say "Integrate with the IBM Cloud Monitoring service to monitor your IBM Cloud Object Storage data in the IBM Cloud"